src/components/SharedComponents/Map/LocationIndicator.tsx

```diff
@@ -18,13 +18,17 @@ const LocationIndicator = ( {
   positionalAccuracy
 }: Props ) => (
   <>
-    <Circle
-      center={{ latitude, longitude }}
-      radius={positionalAccuracy}
-      strokeWidth={2}
-      strokeColor={colors.inatGreen}
-      fillColor="rgba( 116, 172, 0, 0.2 )"
-    />
```

Comment:
Since this conditional render is a functional change, would you mind describing how this impacts the rendering of the LocationIndicator component?

Also this is a nit, but the codebase tends to prefer `{!!positionalAccuracy && <Circle />}` for boolean conditional renders.

Reply:
> Since this conditional render is a functional change, would you mind describing how this impacts the rendering of the LocationIndicator component?

Yep, I tried to articulate it in the pull request description, but I'll add more details to the description now.

> Also this is a nit, but the codebase tends to prefer `{!!positionalAccuracy && <Circle />}` for boolean conditional renders.

Pushed a commit for this